### PR TITLE
[4.0] Display of counts in tables

### PR DESCRIPTION
**Pull request by brianteeman**
pr for #26051

A can be seen in the screenshots below there are two different styles for counts
### com_users groups
image

### com_content categories
image

### com_menus manage
image

### Summary of Changes
banners->clients
smart search > maps
menus
tags
users > groups
workflows

### Testing Instructions
most can be tested with patchtester but smart search maps has a css change so needs npm i

### No bikeshedding
This PR is ONLY about consistency - anything else is off topic and will be ignored. (and hopefully deleted)

### CODE REVIEW

File: administrator/components/com_workflow/tmpl/workflows/default.php [hunk 1]
```diff
@@ -147,15 +147,15 @@
...
+								<td class="text-center btns d-none d-md-table-cell itemnumber">
+									<a class="btn <?php echo ($item->count_states > 0) ? 'btn-warning' : 'btn-secondary'; ?>" title="<?php echo Text::_('COM_WORKFLOW_COUNT_STAGES', true); ?>" href="<?php echo Route::_('index.php?option=com_workflow&view=stages&workflow_id=' . (int) $item->id . '&extension=' . $extension); ?>">
 										<?php echo $item->count_states; ?></a>
 								</td>
-								<td class="text-center btns  d-none d-md-table-cell">
-									<a class="badge <?php echo ($item->count_transitions > 0) ? 'badge-info' : 'badge-secondary'; ?>" title="<?php echo Text::_('COM_WORKFLOW_COUNT_TRANSITIONS', true); ?>" href="<?php echo Route::_('index.php?option=com_workflow&view=transitions&workflow_id=' . (int) $item->id . '&extension=' . $extension); ?>">
+								<td class="text-center btns d-none d-md-table-cel itemnumberl">
```
Comment (Quy) [inline administrator/components/com_workflow/tmpl/workflows/default.php]:
```suggestion
								<td class="text-center btns d-none d-md-table-cell itemnumber">
```

File: administrator/components/com_banners/tmpl/clients/default.php [hunk 2]
```diff
@@ -120,20 +120,20 @@
...
 									</td>
-									<td class="center btns d-none d-lg-table-cell">
-										<a class="badge <?php echo ($item->count_archived > 0) ? 'badge-info' : 'badge-secondary'; ?>" href="<?php echo Route::_('index.php?option=com_banners&view=banners&filter[client_id]=' . (int) $item->id . '&filter[published]=2'); ?>">
+									<td class="center btns d-none d-lg-table-cell itemnumber">
+										<a class="btn <?php echo ($item->count_archived > 0) ? 'btn-info' : 'btn-secondary'; ?>" href="<?php echo Route::_('index.php?option=com_banners&view=banners&filter[client_id]=' . (int) $item->id . '&filter[published]=2'); ?>">
 											<?php echo $item->count_archived; ?></a>
 									</td>
-									<td class="center btns d-none d-lg-table-cell">
-										<a class="badge <?php echo ($item->count_trashed > 0) ? 'badge-inverse' : 'badge-secondary'; ?>" href="<?php echo Route::_('index.php?option=com_banners&view=banners&filter[client_id]=' . (int) $item->id . '&filter[published]=-2'); ?>">
+									<td class="center btns d-none d-lg-table-cell itemnumber	">
```
Comment (Quy) [inline administrator/components/com_banners/tmpl/clients/default.php]:
```suggestion
									<td class="center btns d-none d-lg-table-cell itemnumber">
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 408891b8b3cac4f55753f3e5170371a743106e96<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26066">issues.joomla.org/tracker/joomla-cms/26066</a>.</sub>

**Comment by amit4106udale** [comment]
I have tested this item :white_check_mark: successfully on 408891b8b3cac4f55753f3e5170371a743106e96<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26066">issues.joomla.org/tracker/joomla-cms/26066</a>.</sub>

**Comment by blueforce** [comment]
I have tested this item :white_check_mark: successfully on 408891b8b3cac4f55753f3e5170371a743106e96<br /><br />work as expected
tested with:
PHP-Version	7.3.8
Webserver	Apache
PHP-Interface für den Webserver	cgi-fcgi
Joomla!-Version	Joomla! 4.0.0-alpha12-dev Development [ Amani ] 19-August-2019 13:40 GMT
Browsererkennung	Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.120 Safari/537.36<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26066">issues.joomla.org/tracker/joomla-cms/26066</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26066">issues.joomla.org/tracker/joomla-cms/26066</a>.</sub>

**Comment by idefax** [comment]
I have tested this item :white_check_mark: successfully on 408891b8b3cac4f55753f3e5170371a743106e96<br /><br />Works!

PHP Version 	7.3.8
Web Server 	Apache
WebServer to PHP Interface 	cgi-fcgi
Joomla! Version 	Joomla! 4.0.0-beta1-dev Development [ Amani ] 17-October-2019 20:21 GMT
User Agent 	Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:69.0) Gecko/20100101 Firefox/69.0 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26066">issues.joomla.org/tracker/joomla-cms/26066</a>.</sub>

**Comment by snehaM26** [comment]
I have tested this item :white_check_mark: successfully on 408891b8b3cac4f55753f3e5170371a743106e96<br /><br />Tested successfully<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26066">issues.joomla.org/tracker/joomla-cms/26066</a>.</sub>

**Comment by HLeithner** [comment]
@brianteeman can you resolve the conflict, after that I can merge it. thx

**Comment by brianteeman** [comment]
@HLeithner conflicts resolved

**Comment by HLeithner** [comment]
thx

**Comment by brianteeman** [comment]
Thanks.

**Pull request merged**